### ISSUE TITLE
Dropdown: Not keyboard accessible

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-js#50]
_From @mikewheaton on June 5, 2016 21:47_

The dropdown should be keyboard accessible, both in its regular view and when displayed as a panel on small screens.

_Copied from original issue: OfficeDev/office-ui-fabric-core#639_

**Comment by chethanht** [comment]
Has this been fixed?

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
